### DISCUSSION TITLE
Error when accessing IPAM - IPAddress

### DISCUSSION THREAD

**Discussion by anders-nc** [netbox-community/netbox#17239] [Help Wanted!]
We recently added a bunch of ip addresses using RestAPI calls, to my dismay it has somehow being corrupted.
I can get eralier ipaddresses directly i.e https://netboxserve/ipam/ip-addresses/1599/ but all the addreses added and the main view get this error.
It also hard for us to do a restore of database since it was yesterday and many other things has been updated since.
(but if we must we must of  course)

Is there any logs we can see exactly whats missing or faulty? or a smart method using RestAPI to find all these ipaddresses and delete/rollback these 

<class 'AttributeError'>

'NoneType' object has no attribute 'get_absolute_url'
Python version: 3.11.2
NetBox version: 4.0.8
Plugins: 
  netbox_bgp: 0.13.2
  netbox_dns: 1.0.5
  netbox_documents: 0.7.0
  netbox_interface_synchronization: 4.0.3
  netbox_otp_plugin: 1.1.0
  netbox_secrets: 2.0.3
  netbox_topology_views: 4.0.0

**Comment by anders-nc** [comment]
I removed all faulty ip address object and can now access IPAM IPADDRESS page ttps://netboxserver/ipam/ip-addresses/
However i'm curious about finding what could have gone wrong here , so if there is a better error log or something it would be nice.
We used RESTAPI calls ( POST method ) via powershell and all seemed good, also i could read via RestAPI it was the webgui that was broken

**Reply by markkuleinio** [reply]
If you can recreate the problem with some specific object, I'm sure the developers are curious about it as well. Let us know if you can reproduce it.
